solutions/task_1.py

```diff
@@ -0,0 +1,62 @@
+class Stars:
+    def __init__(self, star_name=None):
```

Comment:
Get rid of default None value

Reply:
It doesn't work without it as when I instantiating Stars() Python is looking for an obligatory argument star_name and crashes each when it fails to find it. On the other hand when I have a default value of star_name I can instantiate Stars() receiving an object with default characteristics.

Reply:
"Get rid of ..." is not "Remove this code: ', star_name=None' "
I don't want to see unnecessary None here.

Reply:
### Read about default keyword arguments!